python/ccf/clients.py

```diff
@@ -54,9 +54,13 @@ class Request:
     headers: dict
 
     def __str__(self):
-        return f"{self.http_verb} {self.path} {self.headers}" + (
-            truncate(f" {self.params}") if self.params is not None else ""
-        )
+        string = f"{self.http_verb} {self.path}"
+        if self.headers:
```

Comment:
Not sure omitting the headers entirely when they're empty is a good thing since we don't name those fields.

Reply:
These formatting changes were in #1429. Think they showed up here briefly due to a merge conflict? Gone now.